hyperopt/fmin.py

```diff
@@ -28,7 +28,7 @@
         'Failed to load cloudpickle, try installing cloudpickle via "pip install '
         'cloudpickle" for enhanced pickling support.'
     )
-    import six.moves.cPickle as pickler
```

Comment:
Is this change intended?

Reply:
I see. You are dropping Python 2 support in this PR too.

Reply:
Python 2 support has actually been dropped earlier (see #876).